### ISSUE TITLE
Color contrast ratio is less than 7:1 for the some of the content in the page

### ISSUE THREAD

**Issue by sagar17sep** [ckeditor/ckeditor-dev#1967]
BUG-

## Provide detailed reproduction steps (if any)

1. Select Color contrast analyser from the WAT tool
2. Open  high contrast and apply both the theme i.e white and black. 

### Expected result
Color contrast ratio should be >=7:1 in high contrast settings.

### Actual result
Color contrast ratio is less than 7:1 for some of the content in the page.

## Other details

* Browser: Edge 41
* OS: Windows 10
* CKEditor version: 4.6.2
* Installed CKEditor plugins: …

**Comment by sagar17sep** [comment]
Please look to the above mentioned issue.

**Comment by sagar17sep** [comment]
Could you please look to the above mentioned issue on high priority.

**Comment by sagar17sep** [comment]
![image](https://user-images.githubusercontent.com/38970470/40957603-8445fb7a-68b3-11e8-9183-0754cbc9d8f6.png)

![image](https://user-images.githubusercontent.com/38970470/40957644-aadb5334-68b3-11e8-9877-c971800aeb26.png)

Please find the updated screenshot for the above mentioned issue.

**Comment by mlewand** [comment]
@sagar17sep it doesn't look anything like our CKEditor default themes (you can check them on [CKEditor 4 demo page](https://ckeditor.com/ckeditor-4/)).

We have verified our skins and we have very good contrast ratio in our default skins on Edge (passes AAA checks).

The situation is worse with the default black high contrast setting on Firefox, as it indeed does have poor contrast  ratio for links, which is being tracked in #2073.

Back to this case, please check your skin customization. Either use the default CKEditor 4 skin or check the issue with your theme provider.
